packages/dm-core/src/hooks/useDocument.tsx

```diff
@@ -51,7 +52,8 @@ interface IUseDocumentReturnType<T> {
 export function useDocument<T>(
   idReference: string,
   depth?: number | undefined,
-  notify: boolean = true
+  notify: boolean = true,
+  throwError: boolean = false
```

Comment:
IMO, "throwError" should be moved into "updateDocument". As not to be confused with throwing error on getting the document.

Reply:
Yes, agree! ✅